partiql-lang/src/main/kotlin/org/partiql/lang/planner/transforms/LogicalToLogicalResolvedVisitorTransform.kt

```diff
@@ -352,25 +354,48 @@ internal data class LogicalToLogicalResolvedVisitorTransform(
 
     override fun transformDmlTarget(node: PartiqlLogical.DmlTarget): PartiqlLogicalResolved.DmlTarget {
         // We only support DML targets that are global variables.
-        val bindingName = BindingName(node.identifier.name.text, node.identifier.case.toBindingCase())
-        val tableUniqueId = when (val resolvedVariable = globals.resolveGlobal(bindingName)) {
+        val bindingId = bindingId(node.identifier)
+        val tableUniqueId = when (val resolvedVariable = globals.resolveGlobal(bindingId)) {
             is GlobalResolutionResult.GlobalVariable -> resolvedVariable.uniqueId
+            is GlobalResolutionResult.NamespacedVariable -> {
+                if (resolvedVariable.getRemainingSteps().isNotEmpty()) {
+                    problemHandler.handleProblem(
+                        Problem(
+                            node.metas.sourceLocationMetaOrUnknown.toProblemLocation(),
+                            PlanningProblemDetails.UndefinedDmlTarget(
+                                bindingId,
+                            )
+                        )
+                    )
+                    "DML Target could not be completely resolved - do not run"
```

Comment:
This message seems geared to the developers applications that embed partiql--not to query authors.  It should be comprehensible to the query author.  i.e. as a query author I don't know what to do with the "do not run" instruction.

Reply:
Would it be presented to the user? Line 384 has very similar wording, but it's used to return a table's "unique id", even when a problem has been emitted to the problem handler. That being said, I'll update the type of problem here.